src/main/react-front-end/src/views/Admin/ClientsInfoComponent.js

```diff
@@ -37,6 +44,41 @@ export default class ClientsInfoComponent extends Component{
 		});
 	}
 
+	// Shows the user a confirmation popup to confirm the update request.
+	enableOrDisableAdmin(event, row){
+		var popupMsg = ""
+		var isAdmin = event.target.checked;
+		if(isAdmin){
+			popupMsg = "Do you want to enable "+ row.firstName + " " + row.lastName+ " as admin.";
+			this.setState({showIsAdminPopup: true, adminChangePopupMsg:popupMsg, targetUser: row.email, isAdmin: true, firstName: row.firstName, lastName: row.lastName});
+		}
+		else{
+			popupMsg = "Do you want to disable "+ row.firstName + " " + row.lastName+ " as admin.";
```

Comment:
Please update this message to
''Please confirm if admin privileges of _______ ________ must be revoked.''

Reply:
Incorporated